docs/en/benchmark/profile_api_server.md

```diff
@@ -0,0 +1,83 @@
+# API Server Performance Test Method
+
+The way to profiling api_server performance is similar to the method for [profiling throughput](./profile_throughput.md). The difference is api_server should be launched successfully before testing.
+
+The evaluation script is `profile_restful_api.py`. Before running it, please install the lmdeploy precompiled package, download the evaluation script and the test dataset:
+
+```shell
+pip install 'lmdeploy[serve]>=0.1.0a0'
+git clone --depth=1 https://github.com/InternLM/lmdeploy
+wget https://huggingface.co/datasets/anon8231489123/ShareGPT_Vicuna_unfiltered/resolve/main/ShareGPT_V3_unfiltered_cleaned_split.json
+```
```

Comment:
cd lmdeploy/benchmark
since all the following commands are executed in this directory.